src/functionalTest/resources/scenarios/RIA-1024-case-submitted-notification-to-case-officer-bradford-save-and-continue-enabled.json

```diff
@@ -1,5 +1,6 @@
 {
-  "description": "RIA-1024 Send case submitted notification to case officer - bradford hearing centre (save and continue enabled)",
+  "description": "RIA-1024 KKK Send case submitted notification to case officer - bradford hearing centre (save and continue enabled)",
```

Comment:
@burale the KKK text in the description should be removed.

Reply:
Removed